client/app/components/examples/dialog.vue

```diff
@@ -0,0 +1,104 @@
+<template lang="pug">
+    extends ../../templates/layout.pug
+
+    block content
+        include ../../templates/category_list.pug
+
+    block dialog
+        div(v-bind:aria-hidden="hidden", role="dialog")
+            div(role='separator', v-on:click="onClose")
+            .wrapper
+                div(role='contentinfo')
+                    header(v-bind:style="headerStyles")
+                        a(v-on:click="onClose", title='close') Close
+                        block dialog_header
```

Comment:
I've got a little doubt about using `block`s, as we probably never need to extend them. I just point it, and let you make a choice.